clients/java/src/main/java/io/zeebe/client/impl/worker/JobWorkerBuilderImpl.java

```diff
@@ -143,6 +152,7 @@ public JobWorker open() {
     ensureGreaterThan("timeout", timeout, 0L);
     ensureNotNullNorEmpty("workerName", workerName);
     ensureGreaterThan("maxJobsActive", maxJobsActive, 0);
+    ensureGreaterThanOrEqualTo("minJobsActiveRatio", minJobsActiveRatio, 0.0f);
```

Comment:
This method needs to be implemented in `io.zeebe.client.impl.command.ArgumentUtil` and imported at the top of this class file in order to use it.